platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheIncompatibleTasksIntegrationTest.groovy

```diff
@@ -221,6 +222,129 @@ class ConfigurationCacheIncompatibleTasksIntegrationTest extends AbstractConfigu
         fixture.assertStateLoaded()
     }
 
+    def "tasks that access project through #providerChain emit no problems when incompatible task is present"() {
+        given:
+        addTasksWithoutProblems()
```

Comment:
💅 I had to look at the source code to check my understanding. I suggest renaming it to something like `addIncompatibleTaskWithoutProblems()`.

Reply:
Makes sense, thanks